Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.messages.in

```diff
@@ -23,6 +23,7 @@
 messages -> RemoteLayerTreeDrawingAreaProxy : DrawingAreaProxy NotRefCounted {
     void SetPreferredFramesPerSecond(unsigned preferredFramesPerSecond)
     void WillCommitLayerTree(WebKit::TransactionID transactionID)
+    void WontCommitLayerTree()
```

Comment:
Slightly odd name. What if DrawingArea::displayDidRefresh() IPC sent a reply instead?

Reply:
It certainly can, it's just a message that isn't specific to RLTDA, so I'll have to add an unnecessary async reply param to WC/CoordinatedGraphics.

Reply:
As discussed offline, changed this message name to `commitLayerTreeNotTriggered` instead.